src/main/java/org/assertj/vavr/internal/Maps.java

```diff
@@ -153,6 +201,28 @@ public static Maps instance() {
         }
     }
 
+    /**
+     * Asserts that the number of entries in the given {@code Map} has the same size as the other array.
+     *
+     * @param info  contains information about the assertion.
+     * @param map   the given {@code Map}.
+     * @param other the group to compare
+     * @throws AssertionError if the given {@code Map} is {@code null}.
+     * @throws AssertionError if the given array is {@code null}.
```

Comment:
Shouldn't this is one and the previous one be an `IllegalArgumentException`?

Reply:
I just followed assertj conventions and rather would like to stick with them. However, this is a valid point - it's about verifying an argument of the method.